smart-cards/src/main/java/gov/va/api/health/smartcards/WellKnownJwksController.java

```diff
@@ -0,0 +1,22 @@
+package gov.va.api.health.smartcards;
+
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    path = "/.well-known/jwks.json",
+    produces = {"application/json"})
```

Comment:
sorry, `fhir+json` can stay. `json+fhir` is deprecated from earlier versions of the FHIR spec